### PR TITLE
Feature/task pause , resume

### PR DESCRIPTION
**Pull request by grogou**

### CODE REVIEW

File: data-provider/class-btm-task-dao.php [hunk 1]
```diff
@@ -336,10 +410,21 @@ public function delete_by_id( $id ){
...
+			$ids_in .= $wpdb->prepare(', %d', $id);
+		}
+
+		$ids_in = ltrim( $ids_in, ', ' );
+
 		$query = $wpdb->prepare('
 			DELETE FROM `' . $this->get_table_name() . '` 
-			WHERE `id` IN ( %s )
-		', implode( ',', $ids ) );
+			WHERE `id` IN ( ' . $ids_in . ' ) AND `status` != "running"
```
Comment (arsendovlatyan) [inline data-provider/class-btm-task-dao.php]: @eriktad "running" must be defined as constant (at least)

File: data-provider/class-btm-task-dao.php [hunk 2]
```diff
@@ -287,6 +287,80 @@ public function mark_as_in_progress( I_BTM_Task $task ){
...
+
+		if( $updated ){
+			return true;
+		}
+
+		return false;
+	}
+
+	/**
+	 * This function should be update only paused tasks to registered tasks
```
Comment (arsendovlatyan) [inline data-provider/class-btm-task-dao.php:330]: @eriktad I'm not sure if function name and description must "force" to change status of paused tasks to 'registered', I guess function name must be like "resume_tasks" and then, in description, we can identify, that new status will be 'registered' after resume.  @zzhovo ?

File: data-provider/class-btm-task-dao.php (hunk 1 above)
Comment (grogou) [reply data-provider/class-btm-task-dao.php]: I'm forgot fix it

File: assets/js/admin.js [hunk 3]
```diff
@@ -9,6 +9,11 @@
 		$( '#date-submit' ).trigger( 'click' )
 	});
 
+	//Bulk verify alert
+	$( '#doaction' ).click(function (e) {
+		alert( 'Are you sure?' );
```
Comment (arsendovlatyan) [inline assets/js/admin.js]: @eriktad is there a way to say "No" and stop the process? I guess you must use confirmation box, not alert.

File: data-provider/class-btm-task-dao.php (hunk 1 above)
Comment (grogou) [reply data-provider/class-btm-task-dao.php]: Fixed

File: assets/js/admin.js (hunk 3 above)
Comment (grogou) [reply assets/js/admin.js]: Fixed

File: data-provider/class-btm-task-dao.php (hunk 2 above)
Comment (grogou) [reply data-provider/class-btm-task-dao.php:330]: Fixed

**Pull request merged**